user_tools/src/spark_rapids_tools/tools/qualification_stats_report.py

```diff
@@ -84,7 +87,14 @@ def _read_csv_files(self) -> None:
 
         rapids_execs_file = self.ctxt.get_value('toolOutput', 'csv', 'execsInformation',
                                                 'fileName')
-        self.execs_df = pd.read_csv(FSUtil.build_path(qual_output_dir, rapids_execs_file))
+        # Load the execs CSV file and drop execs that have no stages or name
+        self.execs_df = (
+            pd.read_csv(FSUtil.build_path(qual_output_dir, rapids_execs_file),
+                        dtype={'Exec Name': str,
+                               'Exec Stages': str,
+                               'Exec Children': str,
```

Comment:
Do we need to explicitly mentioned type for  `Exec Children` ? I thought it would be inferred as string.

Reply:
It is going to be cosnidered float if it is all empty.
So, I decided to set it as well just in case it would be used later in the report.